db/institutions.yml

```diff
@@ -738,6 +728,7 @@ spallationneutronsource:
     - SNS
   city: Oak Ridge
   country: USA
+  departments:
```

Comment:
You are not allowed to have null entries.   the schema for departments is a dictionary, so ``departments: {}`` might work.

This may not be why the tests are failing though if other entries don't have departments.  You can check by clicking on "details" on the failing travis build.